docs/rules/prefer-starts-ends-with.md

```diff
@@ -0,0 +1,19 @@
+# Prefer String#startsWidth & String#endsWidth over more complex alternatives
+
+To know whether a string starts or ends, you have several methods, such as `string.indexOf('foo') === 0` or using regexes with `/^foo/` or `/foo$/`. ES2015 introduces simpler alternatives named `.startsWith` and `.endsWith`. This rule enforces the use of those whenever possible.
```

Comment:
.startsWith => String#startsWith
.endsWith => String#endsWith

and linkify them to their MDN page.
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith